bundles/org.openhab.binding.shelly/README.md

```diff
@@ -0,0 +1,514 @@
+| shellyem           | Shelly EM with integrated Power Meter                  |
+| shelly2-relay      | Shelly Double Relay Switch in relay mode               |
+| shelly2-roller     | Shelly2 in Roller Mode                                 |
+| shelly25-relay     | Shelly 2.5 in Relay Switch                             |
+| shelly25-roller    | Shelly 2.5 in Roller Mode                              |
+| shelly4pro         | Shelly 4x Relay Switch                                 |
+| shellydimmer       | Shelly Dimmer                                          |
+| shellyplugs        | Shelly Plug-S                                          |
+| shellyplug         | Shelly Plug                                            |
+| shellyrgbw2        | Shelly RGB Controller                                  |
+| shellybulb         | Shelly Bulb in Color or White Mode                     |
+| shellyht           | Shelly Sensor (temp+humidity)                          |
+| shellyflood        | Shelly Flood Sensor                                    |
+| shellysmoke        | Shelly Smoke Sensor                                    |
+| shellysense        | Shelly Motion and IR Controller                        |
+| shellyunknown      | Marks password protected or unknown devices            |
+
+## Firmware
+
+To utilize all features the binding requires firmware version 1.5.2 or newer.
+This should be available for all devices.
+Older versions work in general, but have impacts to functionality (e.g. no events for battery powered devices).
+
+The binding displays a WARNING if the firmware is older.
+It also informs you when an update is available.
+Use the device's web ui or the Shelly App to perform the update.
+
+## Discovery
+
+The binding uses mDNS to discovery the Shelly devices.
+They periodically announce their presence, which can be used by the binding to find them on the local network and fetch their IP address.
+The binding will then use the Shelly REST API (HTTP) to discover device capabilities, read status and control the device.
+In addition event callbacks will be used to support battery powered devices.
+
+Make sure to wake-up battery powered devices (press the button inside the device) so they show up on the network.
+Sometime you need to run the manual discovery multiple times until you see all your devices.
+
+Important: The IP address shouldn't change after the device is added as a new thing in openHAB.
+This could be achieved by
+
+- assigning a static IP address or
+- use DHCP and setup the router to assign always the same ip address to the device
+
+### Password protected devices
+
+The Shelly Apps allow to protect device configuration by user id+password, which is  supported by the binding.
+If you are using password protected  devices you need to configure user id and password. 
+A special thing is created if the access to the device's APU cannot be initialized.
```

Comment:
```suggestion
```

Reply:
Replaced by:
The Shelly Apps allow to protect device configuration by user id and password.
In this case you need to configure user id and password in the Thing Configuation. 
Proceed the following steps

- Accept the discovered device from the Inbox
- You'll be prompted for credentials, set user id and password, accept
- The Thing gets re-initialized and the Thing Type is adjusted
- On success the channels are updated, otherwise you need to change credentials.

For those devices the Thing Type "shellydevice" is used.

Reply:
I really meant to simply delete that sentence - this does not belong here.
If such a device is discovered and approved by the user, the following steps should be obvious and not specific to this binding, so there should not be any specific information here.